code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -371,6 +371,35 @@
 		else
 			. += "No operatives detected within scanning range."
 
+/obj/item/pinpointer/operative_nad
```

Comment:
Why not make it inherit "/obj/item/pinpointer/operative"? From what i see the "scan_for_ops" is the same and it would be useful to have their "examine()".